src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -287,6 +287,39 @@ public bool CanProcessResult(object result)
             return result != null;
         }
 
+        internal static void StripExpandoFunctions(object obj)
```

Comment:
We already have the FilteredExpandoObjectConverter that is used on the other response path to do this. Can we reuse that logic to handle on this path as well?

Reply:
No, because the FilteredExpandoObjectConverter converts to json (which would convert byte[] to base 64 string and change the content resolution)

Reply:
This is why it's ugly - it probably makes sense to try and do this at the node level, before edge converts to Expando - but since this is going away I'd rather do the targeted fix with minimal behavior change.

Reply:
Yes, doing this in the node adapter layer would accomplish the main goal of doing this stripping in one place. If we did that, it would fix your bug, and we could get rid of FilteredExpandoObjectConverter?